packages/flutter/test/material/input_chip_test.dart

```diff
@@ -370,32 +368,37 @@ void main() {
   });
 
   testWidgets('Material3 - Input chip has correct selected color when enabled', (WidgetTester tester) async {
-    final ChipThemeData material3ChipDefaults = ThemeData(useMaterial3: true).chipTheme;
+    final ThemeData theme = ThemeData(useMaterial3: true);
+    final ChipThemeData material3ChipDefaults = theme.chipTheme;
     await pumpCheckmarkChip(
       tester,
       chip: selectedInputChip(enabled: true),
-      useMaterial3: true,
+      theme: theme,
     );
 
     final RenderBox materialBox = getMaterialBox(tester);
     expect(materialBox, paints..rrect(color: material3ChipDefaults.backgroundColor));
```

Comment:
This doesn't look right. `material3ChipDefaults.backgroundColor` is a `null` value. 

`ThemeData(useMaterial3: true).chipTheme` is might be incorrect here.

Reply:
Nice catch! 
I investigated a bit and it looks like it could be a faulty implementation of the test, as `paint.rrect(color: null)` passes, and `theme.chipTheme` has all values as null by default. So that expect never had proper non null values to check.
I've updated the test to check for the correct values in the theme colorScheme, but I had to also update how the default input chip in the test is built. The `enabled` param was not being used properly. Regardless the value of `enabled` the chip always had the MaterialState.disabled because it didn't have any of the callbacks (onPressed, onSelected, onDeleted).

I've checked if `filter_chip_test` had the same issue, but it doesn't look like it. The utility chip builder used there doesn't have `enabled` as a parameter.

Changes in 1ab12f1